ee/session_recordings/ai/error_clustering.py

```diff
@@ -76,12 +76,12 @@ def construct_response(df: pd.DataFrame, team: Team, user: User):
     clusters = []
     for cluster, rows in df.groupby("cluster"):
         session_ids = rows["session_id"].unique()
-        sample = rows.sample(n=1)[["session_id", "input"]].rename(columns={"input": "error"}).to_dict("records")
+        sample = rows.sample(n=1)[["session_id", "input"]].rename(columns={"input": "error"}).to_dict("records")[0]
```

Comment:
silly check do we know we have a non-empty thing to get `[0]` from at this point?

Reply:
Yep good question. The cluster will only be generated if it meets the `MIN_SAMPLES` so we know that each cluster has _at least_ 10 rows